src/ctypes/ldouble.mli

```diff
@@ -0,0 +1,258 @@
+(*
```

Comment:
A couple of changes to the naming and module organization would bring this in line with ctypes conventions.

First, calling the module `LDouble` rather than `Ldouble` would match `ULong`, `UInt32`, etc.

Second, it'd be better to have `LDouble.Complex` at the same level as `Ldouble` rather than nested underneath; it could perhaps be called `ComplexL` or somesuch.

(I don't have a strong preference as to whether `LDouble` and `Complex` live at the top level (like `Int32` and `Int64`) or under a parent module such as `Floats` (like `Unsigned.ULong`)).